lib/engine/g_1856/share_pool.rb

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+require_relative '../share_pool'
+
+module Engine
+  module G1856
+    class SharePool < SharePool
+      def buy_shares(entity, shares, exchange: nil, exchange_price: nil, swap: nil)
```

Comment:
this looks copy and pasted what did you change?

Reply:
Lines 62-63 and 104 are changed
`buy_shares` treats `:escrow` corporations similarly to `:incremental` capitalization corporations when invoking `transfer_shares`

`transfer_shares` invokes `escrow_share_buy!` on the corporation if appropriate, this relates to the response to your comment on `corporation.rb` I have yet to finish